### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ikhoon**
- Brave 5.13.10 -> 5.13.11
- Brotli4J 1.7.1 -> 1.8.0
- Dropwizard Metrics 4.2.10 -> 4.2.12
- gRPC-Java 1.48.0 -> 1.49.0
- Jackson 2.13.3 -> 2.13.4
- Logback 1.2.11 -> 1.3.0
- Micrometer 1.9.2 -> 1.9.3
- Reactor 3.4.21 -> 3.4.22
- Sangria 3.0.1 -> 3.2.0
- Scala 3.1.3 -> 3.2.0
- Spring Boot 2.7.2 -> 2.7.3
- Build
  - Dagger 2.43.1 -> 2.43.2
  - Error Prone 2.14.0 -> 2.15.0
  - gax-grpc 2.18.7 -> 2.19.0
  - graphql-kotlin 6.1.0 -> 6.2.2
  - JUnit5 5.8.2 -> 5.9.2
  - JMH plugin 0.6.6 -> 0.6.7
  - Mockito 4.6.1 -> 4.7.0

### CODE REVIEW

**Comment by jrhee17** [comment]
It seems like there is a genuine leak when http2 preface are sent for some reason 🤔  (perhaps related to the netty upgrade)
Taking a closer look

File: dependencies.toml [hunk 1]
```diff
@@ -17,41 +17,41 @@ jetty93 = '9.3.30.v20211001'
...
+dropwizard-metrics = { module = "io.dropwizard.metrics:metrics-bom", version = "4.2.12" }
 # Ensure that we use the same Protobuf version as what gRPC depends on.
 # See: https://github.com/grpc/grpc-java/blob/master/gradle/libs.versions.toml
 #      (Switch to the right tag and look for "protobuf".)
-grpc-java = { module = "io.grpc:grpc-bom", version = "1.48.0" }
-jackson = { module = "com.fasterxml.jackson:jackson-bom", version = "2.13.3" }
+grpc-java = { module = "io.grpc:grpc-bom", version = "1.49.0" }
+jackson = { module = "com.fasterxml.jackson:jackson-bom", version = "2.13.4" }
 junit5 = { module = "org.junit:junit-bom", version = "5.9.0" }
 netty = { module = "io.netty:netty-bom", version = "4.1.79.Final" }
```
Comment (ikhoon) [inline dependencies.toml:56]: The Netty 4.1.80 version is revered because it has a leak in `HttpClientCodec`.
https://github.com/netty/netty/pull/12762

Comment (jrhee17) [reply dependencies.toml:56]: Nice find 👍

File: dependencies.toml [hunk 2]
```diff
@@ -17,41 +17,41 @@ jetty93 = '9.3.30.v20211001'
...
 spring-boot1 = "1.5.22.RELEASE"
-spring-boot2 = "2.7.2"
+spring-boot2 = "2.7.3"
 tomcat8 = "8.5.81"
 tomcat9 = "9.0.65"
 
 [boms]
-brave = { module = "io.zipkin.brave:brave-bom", version = "5.13.10" }
-dropwizard-metrics = { module = "io.dropwizard.metrics:metrics-bom", version = "4.2.10" }
+brave = { module = "io.zipkin.brave:brave-bom", version = "5.13.11" }
```
Comment (ikhoon) [inline dependencies.toml:48]: It was a hassle to find the location of versions back and forth in this file while upgrading versions.
I'd like to migrate embedded dependency versions into the `[versions]` table.
In addition to upgrading the versions, we can easily summarize the new dependencies in the PR description.

**Review by jrhee17 — APPROVED** [review]
Changes looks good to me 👍 Thanks @ikhoon ! 🙇 👍 🙇

**Review by minwoox — APPROVED** [review]
Thanks! 👍

**Pull request merged**